src/frontend/screens/Library/components/InstallModal/SideloadDialog/Warning/index.tsx

```diff
@@ -1,39 +1,39 @@
 import React, { useState } from 'react'
-import { AlertCard, Typography, Images } from '@hyperplay/ui'
-
+import { AlertCard, Images, Typography } from '@hyperplay/ui'
+import { Link } from 'react-router-dom'
 import styles from './index.module.scss'
 import { t } from 'i18next'
 
-const { Button } = Typography
-const { WarningIcon } = Images
-
 export default function SideloadDialogWarning() {
   const [isOpen, setIsOpen] = useState(true)
 
   return (
     <div className={styles.container}>
       {isOpen ? (
         <AlertCard
-          icon={<Images.WarningIcon />}
+          icon={<Images.AlertOctagon />}
           style={{ maxWidth: 670 }}
           onClose={() => setIsOpen(!isOpen)}
-          variant="warning"
+          variant="error"
           title={t('sideload.warningTitle', 'Important')}
           message={t(
             'sideload.warningMessage',
             'Side-loading a game is a feature intended for developers and experienced power users. Side-loading an installer that contains malware could compromise your computer and lead to theft of funds.'
           )}
         />
       ) : (
-        <div
-          className={styles.titleContainer}
-          onClick={() => setIsOpen(!isOpen)}
-        >
-          <WarningIcon />
-          <Button className={styles.title}>
-            {t('sideload.warningTitle', 'Important')}
-          </Button>
-        </div>
+        <a className={styles.warningAlert}>
+          <Images.AlertOctagon className={styles.warningAlertIcon} />
+          <Typography.Button>
+            <Link
+              to="#"
+              className={styles.warningAlertLink}
+              onClick={() => setIsOpen(!isOpen)}
+            >
+              {t('sideload.warningMessageLink', 'Important')}
```

Comment:
```suggestion
              {t('sideload.warningTitle', 'Important')}
```

With this, you'll preserve the existing translations for this key